### PR TITLE
TEM-62: orchestrator integration

### PR DESCRIPTION
**Pull request by bobheadxi**
## :construction_worker: Purpose

Spin up network when creating network entry

## :rocket: Changes

Removed local node peer requirement
Removed API_URL requirement
Renamed Temporal/grpc to Temporal/grpc-clients

## :warning: Breaking Changes

Nothing significant, just looser API requirements

### CODE REVIEW

**Comment by postables** [comment]
we should disable tns tests for now since they appear to be causing intermitent issues with travis

File: api/routes_rtfsp.go [hunk 1]
```diff
@@ -530,105 +531,118 @@ func (api *API) publishDetailedIPNSToHostedIPFSNetwork(c *gin.Context) {
...
-	bPeers, exists := c.GetPostFormArray("bootstrap_peers")
-	if !exists {
-		FailWithMissingField(c, "bootstrap_peers")
-		return
-	}
-	nodeAddresses, exists := c.GetPostFormArray("local_node_addresses")
+	networkName, exists := c.GetPostForm("network_name")
 	if !exists {
-		FailWithMissingField(c, "local_node_addresses")
+		FailWithBadRequest(c, "network_name")
```
Comment (postables) [inline api/routes_rtfsp.go]: In order to communicate the error is due to a missing field shouldn't it be:
```suggestion
		FailWithMissingField(c, "network_name")
```

File: api/routes_rtfsp.go [hunk 2]
```diff
@@ -530,105 +531,118 @@ func (api *API) publishDetailedIPNSToHostedIPFSNetwork(c *gin.Context) {
...
+			"users":        network.Users,
+		},
+	})
+}
+
+func (api *API) stopIPFSPrivateNetwork(c *gin.Context) {
+	username := GetAuthenticatedUserFromContext(c)
+	networkName, exists := c.GetPostForm("network_name")
+	if !exists {
+		FailWithBadRequest(c, "network_name")
```
Comment (postables) [inline api/routes_rtfsp.go]:
```suggestion
		FailWithMissingField(c, "network_name")
```

**Comment by bobheadxi** [comment]
I'll be holding off on this PR until the database updates go through 👍

**Pull request merged**